src/index.js

```diff
@@ -27,6 +27,11 @@ class ScrollableList extends Component {
 
   componentWillReceiveProps(props) {
     if (this.props.data !== props.data) {
+
+      if (props.data == null) {
```

Comment:
Instead of adding this extra check, I would do this: https://github.com/nachoaIvarez/react-native-scrollable-list/pull/7/files#r68987416